src/Uno.UWP/Globalization/ApplicationLanguages.cs

```diff
@@ -1,29 +1,93 @@
 ﻿using System;
 using System.Globalization;
 using System.Linq;
-using System.Reflection;
 using System.Text.RegularExpressions;
+using Windows.Storage;
 using Uno.Foundation.Logging;
 using Uno.UI;
+using Uno;
 
 namespace Windows.Globalization
 {
 	public static partial class ApplicationLanguages
 	{
-		private static string _primaryLanguageOverride;
+		private static string _primaryLanguageOverride = string.Empty;
+		private const string PrimaryLanguageOverrideSettingKey = "__Uno.PrimaryLanguageOverride";
 
 		static ApplicationLanguages()
 		{
+#if !NET461
```

Comment:
If setting localsettings not supported for net461, then we should have the same condition for where the PrimaryLanguageOverride is persisted in the static PrimaryLanguageOverride property.